routing/dht/routing.go

```diff
@@ -84,7 +84,10 @@ func (dht *IpfsDHT) GetValue(ctx context.Context, key key.Key) ([]byte, error) {
 	ctx, cancel := context.WithTimeout(ctx, time.Minute)
 	defer cancel()
 
-	vals, err := dht.GetValues(ctx, key, 16)
+	// retrieve a majority of the expected record count
+	majority := (KValue / 2) + 1
```

Comment:
I'm confident that we are fine lowering the value from 16 to 11, every time a query is made, if a node requests K, and receives values A B B A C B, and A is the most valid record, then the node making the query will 'correct' all the nodes who did not send A by sending them a STORE rpc with the value A. Because of this, the probabilities of getting the freshest record are greatly improved.

All of this is assuming that we change K from 10 to 20 (as recommended by the spec, which is also the reason I assumed it was 20 this whole time)